houston/queries.go

```diff
@@ -426,4 +426,17 @@ var (
         	log: message
       }
     }`
+	DeploymentInfoRequest = `
+	query DeploymentInfo {
+		deploymentConfig {
+			airflowImages {
+			version
+			distro
+			isDefault
+		}
+		airflowVersions
```

Comment:
@andriisoldatenko we don't use `airflowVersions` or `defaultAirflowChartVersion` do we? GraphQL lets you omit any data you don't need to reduce response size.

Reply:
nice catch! one sec